gsadjust/GSadjust.py

```diff
@@ -120,26 +120,37 @@
 from data_correction import time_correction
 from data_export import export_metadata, export_summary, export_data
 # GSadjust modules
-from data_import import read_csv, read_burris, read_cg6, read_cg6tsoft, read_scintrex, import_abs_g_complete, \
-    import_abs_g_simple
+from data_import import (
+    read_csv, read_burris, read_cg6, read_cg6tsoft, read_scintrex, 
+    import_abs_g_complete, import_abs_g_simple
+)
 from data_objects import Datum, Tare, ChannelList, Delta, SimpleDelta
 from gsa_plots import plot_loop_animation
 from gsa_plots import plot_network_graph, plot_compare_datum_to_adjusted, plot_adjust_residual_histogram
-from gui_objects import AddDatumFromList, CoordinatesTable
-from gui_objects import FigureDatumComparisonTimeSeries
-from gui_objects import GravityChangeTable, TideCorrectionDialog, TideCoordinatesDialog, ApplyTimeCorrection
-from gui_objects import VerticalGradientDialog, AddTareDialog, MeterType, LoopTimeThresholdDialog, Overwrite
+from gui_objects import (
+    AddDatumFromList, CoordinatesTable, 
+    FigureDatumComparisonTimeSeries,
+    GravityChangeTable, TideCorrectionDialog, TideCoordinatesDialog, ApplyTimeCorrection,
+    VerticalGradientDialog, AddTareDialog, MeterType, LoopTimeThresholdDialog, Overwrite,
+    show_message, SelectAbsg, AdjustOptions, LoopOptions
+)
 from menus import Menus
-from pyqt_models import BurrisTableModel, ScintrexTableModel
-from pyqt_models import ObsTreeModel, TareTableModel
-from pyqt_models import ObsTreeStation, ObsTreeLoop, ObsTreeSurvey
+from pyqt_models import (
+    BurrisTableModel, ScintrexTableModel, ObsTreeModel, TareTableModel,
+    ObsTreeStation, ObsTreeLoop, ObsTreeSurvey
+)
 from tab_data import TabData
 from tab_drift import TabDrift
 from tab_network import TabAdjust
 from tide_correction import tide_correction_agnew, tide_correction_meter
 from utils import *
 
 
+DOWN = 1
```

Comment:
I moved these constants off the window object, since they're not really related to it. If you find yourself creating a lot of these you can also put them in a file `constants.py` for example, and then import them as needed.